### PR TITLE
Update Tendermint definitions

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: chain/tendermint/proto/codec.proto [hunk 1]
```diff
@@ -9,250 +9,279 @@ import "google/protobuf/descriptor.proto";
...
 
-  SIGNED_MSG_TYPE_UNKNOWN = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
+  SIGNED_MSG_TYPE_UNKNOWN   = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
   // Votes
-  SIGNED_MSG_TYPE_PREVOTE = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
+  SIGNED_MSG_TYPE_PREVOTE   = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
   SIGNED_MSG_TYPE_PRECOMMIT = 2 [(gogoproto.enumvalue_customname) = "PrecommitType"];
   // Proposals
-  SIGNED_MSG_TYPE_PROPOSAL = 3 [(gogoproto.enumvalue_customname) = "ProposalType"];
+  SIGNED_MSG_TYPE_PROPOSAL  = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
```
Comment (roignpar) [inline chain/tendermint/proto/codec.proto:20]: why 32?

File: chain/tendermint/src/chain.rs [hunk 2]
```diff
@@ -258,13 +258,15 @@ impl FirehoseMapperTrait<Chain> for FirehoseMapper {
             .as_ref()
             .expect("block payload information should always be present");
 
+        // info!(_logger, "block: {:#?}", response.block.new_block);
```
Comment (roignpar) [inline chain/tendermint/src/chain.rs:261]: do we need to keep commented out code?

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 3]
```diff
@@ -62,10 +10,26 @@ pub struct Block {
...
+    #[prost(int64, tag = "2")]
+    pub max_gas: i64,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Commit {
     /// must be > 0
-    #[prost(uint64, tag = "1")]
-    pub height: u64,
+    #[prost(int64, tag = "1")]
+    pub height: i64,
```
Comment (roignpar) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:32]: if it must be `> 0`, wouldn't it make sense to keep it as `uint64`?

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 4]
```diff
@@ -331,41 +326,79 @@ pub struct ResponseDeliverTx {
...
-/// EventValidatorSetUpdates
 #[derive(Clone, PartialEq, ::prost::Message)]
-pub struct EventValidatorSetUpdates {
+pub struct ValidatorParams {
+    #[prost(string, repeated, tag = "1")]
+    pub pub_key_types: ::prost::alloc::vec::Vec<::prost::alloc::string::String>,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct ValidatorSet {
+    /// cannot be empty of nil
```
Comment (roignpar) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:379]: "or"?

File: chain/tendermint/src/chain.rs [hunk 5]
```diff
@@ -18,7 +18,7 @@ use graph::{
     log::factory::{ComponentLoggerConfig, ElasticComponentLoggerConfig},
     prelude::{
         async_trait, o, BlockNumber, ChainStore, Error, Logger, LoggerFactory, StopwatchMetrics,
-        SubgraphStore,
+        SubgraphStore, info
```
Comment (roignpar) [inline chain/tendermint/src/chain.rs:21]: `info` is never used.

File: chain/tendermint/proto/codec.proto [hunk 6]
```diff
@@ -9,250 +9,279 @@ import "google/protobuf/descriptor.proto";
...
+  PublicKey pub_key = 2 [(gogoproto.nullable) = false];
+  int64     power   = 3;
+}
+
+message VersionParams {
+  option (gogoproto.populate) = true;
+  option (gogoproto.equal)    = true;
+
+  uint64 app_version = 1;
 }
```
Comment (roignpar) [inline chain/tendermint/proto/codec.proto:287]: AFAIK all the `gogoproto` annotations have no value in a rust context.

**Comment by roignpar** [comment]
The rust files should be formatted (`rustfmt`).

File: chain/tendermint/proto/codec.proto (hunk 1 above)
Comment (patiee) [reply chain/tendermint/proto/codec.proto:20]: https://github.com/tendermint/tendermint/blob/85870def7b628effad73af942e638bbddf2ba8fd/proto/tendermint/types/types.proto#L34

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs (hunk 3 above)
Comment (patiee) [reply chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:32]: it's autogenerated from proto, I can change it

File: chain/tendermint/proto/codec.proto (hunk 6 above)
Comment (patiee) [reply chain/tendermint/proto/codec.proto:287]: this file is just copied, comments to proto please leave on this PR: https://github.com/figment-networks/tendermint-protobuf-def/pull/7

**Pull request merged**